### PR TITLE
Maintain combined_fees.csv that can be pulled in a single request by defilytica.

### PR DESCRIPTION
**Pull request by Tritium-VLK**

### CODE REVIEW

**Comment by Tritium-VLK** [comment]
Will demonstrate once the fee allocator is working again after https://github.com/BalancerMaxis/multisig-ops/pull/889 is merged.

**Comment by Tritium-VLK** [comment]
Example result:
https://github.com/BalancerMaxis/protocol_fee_allocator/pull/160

**Comment by Xeonus** [comment]
@Tritium-VLK can you adjust the header and replace `date string` with `date_string` ? Thanks

**Comment by Xeonus** [comment]
@Tritium-VLK - all functionality has been tested and data can be successfully ingested. Feel free to merge and make sure we get a combined run after each fee run. Can do after this weeks fee processing to make sure we don't have bugs / interference. Thanks a lot.

**Comment by Tritium-VLK** [comment]
@Xeonus Someone needs to approve first, but feel free to approve and merge.

**Comment by Xeonus** [comment]
@Tritium-VLK can you first fix the header bugs when you find time? Then I can approve

**Comment by Tritium-VLK** [comment]
@Xeonus did we agree to leave the header as is and do the transformatio on the javascript?  If so please approve and merge. If not I can take some time later today and try to override the header as it is written.

**Review by Xeonus — APPROVED** [review]
Approving for now as I can handle "bad" headers easily

File: generate_merged_allocation_csv.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
+import os
+import pandas as pd
+
+# This script combines all the incentives_ csvs in the directory specified into a single csv.
+# It adds a column that includes the important part of the string from the file it came from in each row of the csv.
+
+
+# Directory containing the CSV files
+directory = 'fee_allocator/allocations'
```
[the remark this answers is not in the document]
Comment (Tritium-VLK) [reply generate_merged_allocation_csv.py:9]: Can you more clearly demonstrate the specific problem/concern you have?

**Pull request merged**